cmd/multi-factor-auth/config/config.toml

```diff
@@ -28,6 +28,7 @@
     RequestTimeInSeconds = 2
     SkipTxUserSigVerify = true
     MaxTransactionsAllowedForSigning = 1000
+    MaxContentLengthAllowed = 100
```

Comment:
since this will apply to all endpoint, i think the default value need to be higher

Reply:
increased to 1000